assets/interfaces/common.ts

```diff
@@ -80,3 +80,13 @@ export interface IOccurStatus {
     qcode: string;
     label: string;
 }
+
+export interface IDateFilters{
+    name: string,
+    default: boolean,
+    query?: Array<{}>
+    filter: Array<['today', 'last_week', 'last_30_days', '']>
```

Comment:
and here the filter names could be different if there will be some client specific ones defined, that should be rather a string